tests/integration/test_end_point.py

```diff
@@ -222,6 +218,124 @@ def test_status_with_database_and_navitia_ko(setup_database):
         assert "navitia_url" in resp
 
 
+def test_status_with_redis_and_rabbitmq_down(setup_database):
+    kirin.redis_client.connection_pool = ConnectionPool(host="wrong_url_status")
+    kirin.rmq_handler = RabbitMQHandler("wrong_rabbit_url", "wrong_exchange", "wrong_exchange_type")
+    assert_call_status_200()
+
+
+def test_health_with_redis_and_rabbitmq_down():
+    kirin.redis_client.connection_pool = ConnectionPool(host="wrong_url_health")
+    kirin.rmq_handler = RabbitMQHandler("wrong_rabbit_url", "wrong_exchange", "wrong_exchange_type")
+    assert_call_health_with_ko_and_503()
+
+
+def test_contributors_with_redis_and_rabbitmq_down(setup_database):
+    kirin.redis_client.connection_pool = ConnectionPool(host="wrong_url_contributors")
+    kirin.rmq_handler = RabbitMQHandler("wrong_rabbit_url", "wrong_exchange", "wrong_exchange_type")
+    assert_call_contributors_200()
+
+
+def test_root_with_redis_and_rabbitmq_down(setup_database):
+    kirin.redis_client.connection_pool = ConnectionPool(host="wrong_url_root")
+    kirin.rmq_handler = RabbitMQHandler("wrong_rabbit_url", "wrong_exchange", "wrong_exchange_type")
+    assert_call_root_200()
+
+
+def test_piv_simple_feed_with_redis_and_rabbitmq_down(setup_database):
+    kirin.redis_client.connection_pool = ConnectionPool()
+    kirin.rmq_handler = RabbitMQHandler("wrong_rabbit_url", "wrong_exchange")
+    from tests.integration.piv_test import _get_stomp_20201022_23187_delayed_5min_fixture
+
+    piv_str = ujson.dumps(_get_stomp_20201022_23187_delayed_5min_fixture())
+    with pytest.raises(ConnectionError):
+        api_post("/piv/{}".format(PIV_CONTRIBUTOR_ID), data=piv_str, check=False)
+
+    assert_call_root_200()
+    assert_call_contributors_200()
+    assert_call_health_with_ko_and_503()
+    assert_call_status_200()
+
+
+def test_gtfsrt_simple_feed_with_redis_and_rabbitmq_down(basic_gtfs_rt_data, setup_database):
+    kirin.redis_client.connection_pool = ConnectionPool()
+    kirin.rmq_handler = RabbitMQHandler("wrong_rabbit_url", "wrong_exchange")
+    tester = app.test_client()
+    with pytest.raises(ConnectionError):
+        tester.post("/gtfs_rt/{}".format(GTFS_CONTRIBUTOR_ID), data=basic_gtfs_rt_data)
+
+    assert_call_root_200()
+    assert_call_contributors_200()
+    assert_call_health_with_ko_and_503()
+    assert_call_status_200()
+
+
+def assert_call_root_200():
```

Comment:
Minor (do it only if you want) : I'd be more specific about the call (same on all).
```suggestion
def assert_get_root_200():
```